src/pudl/helpers.py

```diff
@@ -893,15 +893,14 @@ def remove_leading_zeros_from_numeric_strings(
         column.
 
     """
-    if col_name in df.columns:
```

Comment:
Did you mean to remove this check? It could fail in cases where it worked before now, if it's being called on dataframes that don't have the column it's trying to fix.

Reply:
Yes -- CG had left a comment about it earlier that I forgot to address. The `if` was a relic from the function's previous form where it only accepted `generator_id`. Right now we want it to fail if you pass it a column that's not in the database rather than just move along. All instances where it's used rn are very intentional about passing it a column that should in fact be in the database.

Reply:
Resolved this by adding `if` statements prior to each time the function is called in the extract modules so that it won't break.